docs/pages/guides/authentication.mdx

```diff
@@ -1940,6 +1942,115 @@ export default function App() {
 </Box>
 {/* End Uber */}
 
+{/* Start Calendly*/}
+<Box
+  name="Calendly"
+  createUrl="https://developer.calendly.com/getting-started"
+  image={ASSETS.calendly}
+>
+| Website                   | Provider  | PKCE      | Auto Discovery |
+| ------------------------- | --------- | --------- | -------------- |
+| [Get Your Config](https://developer.calendly.com/getting-started) | OAuth 2.0 | Supported | Not Available  |
+- `redirectUri` requires 2 slashes (`://`).
+- Example redirectUri:
+  - Standalone / development build: `myapp://*`
+  - Web: `https://yourwebsite.com/*`
+<Tabs tabs={["Auth Code"]}>
+
+<Tab>
+{/* prettier-ignore */}
```

Comment:
```suggestion
<Tab>

{/* prettier-ignore */}
```

Let's add the space so that Prettier does not auto-format the code snippet below.